src/mudlet.cpp

```diff
@@ -3685,7 +3690,7 @@ void mudlet::doAutoLogin(const QString& profile_name)
 
     // load an old profile if there is any
     // PLACEMARKER: Host creation (2) - autoload case
```

Comment:
Please revise the number down now that `// PLACEMARKER: Host creation (1) ...` has gone away.

Reply:
I still don't see the point of these placemakers. Revised, though.

Reply:
They were/are marking all the places where a profile/`Host` can be created (and there are some for destruction) - which is useful to check that all of them are passed through/tested in some situations (as we have had cases where one execution path showed a bug that another one didn't) and putting breakpoints on them is one technique I have used to debug things in those cases.

Reply:
Modern IDEs have a "find all usages of this function" feature which does the same, but better: it never forgets to updates comments.

Reply:
I just like the Qt TODO plugin as it makes it very quick and simple to jump to previously commented/marked places in the whole project...

![Screenshot_20190812_164548](https://user-images.githubusercontent.com/6163092/62878443-fc3d7500-bd20-11e9-9f9d-d7acc88cd250.png)

![Screenshot_20190812_164718](https://user-images.githubusercontent.com/6163092/62878442-fba4de80-bd20-11e9-9ca1-58538107d568.png)

Reply:
Yes but, doing these things requires manual work to keep it up to date - which can be wrong! So it has a low reliability score - thus I'd ignore it and only trust the source...